app/javascript/components/App.js

```diff
@@ -22,12 +22,19 @@ class App extends React.Component {
     this.state = {
       parties: MockParties,
       items: MockItems,
+      //mock party id
+      party_id: 0,
     };
   }
 
   createNewEvent = (newEvent) => {
     console.log(newEvent);
-  }
+    this.setState({ party_id: (this.state.party_id += 1) });
```

Comment:
this logic seems to be dependent on mock data and not data that will be actually represented in the database